src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsService.kt

```diff
@@ -1113,7 +1113,7 @@ private fun OutcomeHistoryDto.toNomisCreateHearingResult(): CreateHearingResultR
     findingCode = toNomisFindingCode(findingCode),
     adjudicatorUsername = getAdjudicatorUsernameForInternalHearingOnly(
       hearing.oicHearingType.name,
-      hearing.outcome!!.adjudicator,
+      hearing.outcome.adjudicator,
```

Comment:
@mikehalmamoj do you not get those annoying IntelliJ Kotlin errors which means you have to do a whole new clean build?